client/setup.py

```diff
@@ -34,13 +33,13 @@ def get_version():
       ],
       extras_require={
           'test': [
-              'pytest==3.0.2',
-              'moto==0.4.27',
-              'twine==1.5.0',
-              'pip==7.1.0',
-              'wheel==0.24.0',
-              'flake8==2.5.0',
-              'responses==0.5.0',
+              'pytest',
+              'moto<3.0.0',
+              'twine',
+              'pip',
+              'wheel',
+              'flake8',
+              'responses',
```

Comment:
Can you pin these to specific versions? Or at least bound them to be `<` some major version?

Reply:
Done!